README.md

```diff
@@ -147,6 +147,10 @@ render_component(
 
     # An optional object which will be used instead of the default renderer
     renderer=None,
+
+    # An optional dictionary of request header information (such as `Accept-language`)
+    # to pass along with the request to the render server
+    headers=None,
```

Comment:
Everything looks fine here.  My only comment is about the naming of this new parameter.  Will this always only be header information?  Or is it possible this may just be "context" information that could come from places other than the headers.